.github/workflows/lint-test.yml

```diff
@@ -206,7 +223,7 @@ jobs:
         TZ: "America/New_York"
       run: |
         echo "$GITHUB_STEP_SUMMARY_HEADER" | sed "s/#name#/Test Summary/" >> $GITHUB_STEP_SUMMARY
-        nox -s test-${{ matrix.python-version }} -- --color=no --cov-report= | tee -a $GITHUB_STEP_SUMMARY
+        nox --force-python ${{ matrix.python-version }} -s test -- --color=no --cov-report= | tee -a $GITHUB_STEP_SUMMARY
```

Comment:
tests currently fail due to the `audioop` deprecation, for now we can just suppress it using
```py
with warnings.catch_warnings():
    warnings.simplefilter("ignore", DeprecationWarning)
    import audioop
```

Reply:
https://github.com/DisnakeDev/disnake/pull/785/commits/d390111b50ceebae672bf71a35cd490b99f304a5